kratos/elements/embedded_nodal_variable_calculation_element_simplex.cpp

```diff
@@ -89,10 +89,12 @@ void EmbeddedNodalVariableCalculationElementSimplex<double>::CalculateLeftHandSi
     // Get the element shape function values from the normalized distance to node 0
     const auto &rN = this->GetDistanceBasedShapeFunctionValues();
 
-    // Compute the Gramm matrix
+    // Compute the Gramm matrix and gradient penalty term
+    const double penalty = rCurrentProcessInfo[GRADIENT_PENALTY_COEFFICIENT];
+    std::vector<double> aux_penalty({penalty, -penalty});
```

Comment:
make this to be an std::array instead of std::vector so that it is allocated in stack

Reply:
Sure :man_facepalming: